### ISSUE TITLE
Notification counter displays incorrect number (mobile)

### ISSUE THREAD

**Issue by whjms** [apxltd/what-bugs#149]
The notification counter says '11'.

Expected: there are 11 entries in the unread notification drawer
Actual: there are 7 entries

I'm guessing this is because the forum adds +1 for each upvote notification but folds them together when displaying them.

![screenshot_20160912-171611](https://cloud.githubusercontent.com/assets/3392687/18456403/a15d0c02-791e-11e6-840c-5714d3f87afa.png)

**Comment by BenLubar** [comment]
> I'm guessing this is because the forum adds +1 for each upvote notification but folds them together when displaying them.

Yep. Those are separate notifications, but they all get cleared at once.
